cherrypy/test/test_states.py

```diff
@@ -110,7 +112,7 @@ def test_0_NormalStateFlow(self):
 
         host = cherrypy.server.socket_host
         port = cherrypy.server.socket_port
-        self.assertRaises(IOError, cherrypy._cpserver.check_port, host, port)
+        self.assertRaises(IOError, portend._check_port, host, port, timeout=0.1)
```

Comment:
Yes, this is definitely bad form. I allowed it since this was for a test and not in the code. It might be better to make that a public function. I didn't want to make it public in portend because it exposes somewhat obscure implementation details. But the need for its usage here and the fact that I've found the need for it elsewhere suggests that I should invest the time and expose a proper function in portend.